File: Selection_sorting.py
# This program is to do selective sorting
print("Enter no of elements in array")

# Getting number.of elements in the array
n = int(input())
a = [0] * (n + 1)

# Running loop to get array elements
for i in range(1, n + 1, 1):
    print("Enter the elements in array" + str(i))
    a[i] = int(input())

# Performing loop to arrange the elements in the ascending order
for i in range(1, n - 1 + 1, 1):
    for j in range(i + 1, n + 1, 1):
        
        # Using 'if loop' to find highest number and arrange it in order
        if a[j] < a[i]:
            m = a[j]
            a[j] = a[i]
            a[i] = m
print("The selection sort")
for i in range(1, n + 1, 1):
    print(a[i])
    
    # Printing the output in ascending order

File: C/Selection_Sorting.c
#include <stdio.h>

int main() {
    int n, i, j, m;
    printf("Enter no of elements in array\n");
    scanf("%d", &n);
    int a[n+1];
    for (i = 1; i <= n; i++) {
        printf("Enter the elements in array%d\n", i);
        scanf("%d", &a[i]);
    }
    for (i = 1; i <= n-1; i++) {
        for (j = i+1; j <= n; j++) {
            if (a[j] < a[i]) {
                m = a[j];
                a[j] = a[i];
                a[i] = m;
            }
        }
    }
    printf("The selection sort\n");
    for (i = 1; i <= n; i++) {
        printf("%d\n", a[i]);
    }
    return 0;
}
